### ISSUE TITLE
Squiz.Commenting.FunctionCommentThrowTag.Missing after update to PHP 7.0.4

### ISSUE THREAD

**Issue by Taffarel** [squizlabs/PHP_CodeSniffer#1031]
Hi guys!!

First I will put a example of my code and then I will try to explain as clear as possible.

``` php
<?php

use MyApp\Exception\MyException;

class MyClass
{
    /**
     * ...
     * @throws MyApp\Exception\MyException
     */
    public function myMethod()
    {
        ...
        throw new MyException;
    }
    ...
}
```

and when I run the phpcs I get the following error:

``` bash
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 63 | ERROR | Missing @throws tag for "MyException" exception
    |       | (Squiz.Commenting.FunctionCommentThrowTag.Missing)
```

This issue start after I update my php 5.6 to 7.0.4

I've already try to update my codesniffer version to the last release and still got the same error.

Other things that I tried was to declare the full namespace

``` php
 throw new MyApp\Exception\MyException;
```

But I think this is redundant because I already declare in the namespace area.

Is there any tip to make this work as before or for every exception I will have to declare something like this:

``` php
<?php

use MyApp\Exception\MyException;

class MyClass
{
    /**
     * ...
     * @throws MyException
     */
    public function myMethod()
    {
        ...
        throw new MyException;
    }
    ...
}
```

**Comment by fdiasdev** [comment]
any thoughts?

**Comment by aik099** [comment]
I wonder why Travis CI, that also uses PHP 7 doesn't pick that up as failing tests.

**Comment by gsherwood** [comment]
In my testing, the sniff performs exactly the same way in PHP 5 and PHP 7.

**Comment by gsherwood** [comment]
This particular sniff has never had any sort of namespace support, so it's just looking for an exact match. If that's not what you do, then this sniff probably isn't useful to you right now.

This is a good feature request for the sniff though.

**Comment by gsherwood** [comment]
This is actually the same issue as #322, so I'll close this one.

**Comment by jrfnl** [comment]
I believe this has been fixed via PR #2322, which will be included in the 3.4.1 release.
